.github/workflows/_hugo.yml

```
@@ -0,0 +1,55 @@
+        with:
+          path: ./public
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v4
+        with:
+          environment: ${{ env.ENVIRONMENT_NAME }}
```

actionlint

input "environment" is not defined in action "actions/deploy-pages@v4". available inputs are "artifact_name", "error_count", "preview", "reporting_interval", "timeout", "token"

(action)